PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,3 +9,15 @@ Describe the technical details of realization provided by you.
 
 ### Additional context (optional)
 Add any other context or screenshots about the feature request here.
+
+### Make sure these boxes are checked:
+- [ ] Check all the changes in github PR - files count (non of them are redundant, have meaningful changes, all are added), if target branch is correct
+- [ ] Check methods and variable namings - it should be self descriptive, no typos
+- [ ] Check you did not introduce breaking changes in API and public models/services.
+- [ ] Respect extensibility - https://community.virtocommerce.com/t/extensibility-basics-the-domain-model-and-persistence-layer-extension/141
+- [ ] Follow DRY and SOLID principles
```

Comment:
Let's add the links to the wiki:
Follow [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and [SOLID](https://en.wikipedia.org/wiki/SOLID) principles